### PR TITLE
[15.0][MIG] stock_move_value_report: Migration to version 15.0

### PR DESCRIPTION
**Pull request by LudLaf**
cc @Tecnativa TT38342

### CODE REVIEW

File: stock_move_value_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To print the report you can either go to:
+
+- *Inventory > Transfrers*
```
Comment (pedrobaeza) [inline stock_move_value_report/README.rst:42]:
```suggestion
- *Inventory > Transfers*
```

File: stock_move_value_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- *Inventory > Reporting > Stock Moves*
+- *Inventory > Reporting > Product Moves*
+
+And then:
+
+#. Switch to list view.
+#. Select records to print.
+#. And then *Print > Operations Cost Value Report*
+
+A pdf preport with the operations linked to these records and their cost will
```
Comment (pedrobaeza) [inline stock_move_value_report/README.rst:54]:
```suggestion
A pdf report with the operations linked to these records and their cost will
```

**Review by pedrobaeza — DISMISSED** [review]
Please test each report. For example, there's no more `stock.inventory` object.

**Comment by LudLaf** [comment]
> Please test each report. For example, there's no more `stock.inventory` object.

All reports have been tested, and the one related to `stock.inventory` has been removed.
Besides, all other requested changes are now fixed.

**Review by carlosdauden — APPROVED** [review]
Code LGTM

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**